readme.txt

```diff
@@ -20,17 +20,24 @@ Learn
 -----
 
 There is no tutorial for SFML.Net, but since it's a binding you can use the C++ resources:
-* The official tutorials (http://www.sfml-dev.org/resources.php#tutorials)
-* The online API documentation (http://www.sfml-dev.org/resources.php#documentation)
+* The official tutorials (http://www.sfml-dev.org/tutorials/)
+* The online API documentation (http://www.sfml-dev.org/documentation/)
 * The community wiki (https://github.com/LaurentGomila/SFML/wiki/)
 * The community forum (http://en.sfml-dev.org/forums/) (or http://fr.sfml-dev.org/forums/ for french people)
 
 Of course, you can also find the SFML.Net API documentation in the SDK.
 
+Building
+--------
+
+To build and use SFML.NET you must have a copy of CSFML. CSFML can be compiled from
+source (https://github.com/LaurentGomila/CSFML/) or downloaded from the offical release
+page (http://www.sfml-dev.org/download/csfml/). Also note that CSFML also relies on main SFML project.
+
```

Comment:
I would add a word about carefully matching the versions of CSFML and SFML.Net binaries.